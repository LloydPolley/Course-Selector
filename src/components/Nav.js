import React from 'react';
import { NavLink } from 'react-router-dom';
import '../styles/nav.css';


const Nav = () => (
    <div className="nav">
        <NavLink to="/" activeClassName="is-active" exact={true}>Dashboard</NavLink>
        <NavLink to="/add" activeClassName="is-active">Add Course</NavLink>
    </div>
)

export default Nav;