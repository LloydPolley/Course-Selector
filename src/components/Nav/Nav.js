import React from 'react';
import { NavLink } from 'react-router-dom';
import '../../styles/nav.css';


const Nav = () => (
    <div className="nav">
        <NavLink to="/" activeClassName="is-active" exact={true}>Dashboard</NavLink>
        <NavLink to="/add" activeClassName="is-active">Add Course</NavLink>
        <NavLink to="/addSchool" activeClassName="is-active">Add School</NavLink>
        <NavLink to="/preview" activeClassName="is-active">Preview</NavLink>
    </div>
)

export default Nav;