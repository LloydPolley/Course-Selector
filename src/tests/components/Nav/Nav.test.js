import React from 'react';
import Nav from '../../../components/Nav/Nav';
import { shallow, mount, render } from 'enzyme';
import '../../../../setupTests';
import toJSON from 'enzyme-to-json';


test('should render nav', () => {
    const wrapper = shallow(<Nav/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
})

