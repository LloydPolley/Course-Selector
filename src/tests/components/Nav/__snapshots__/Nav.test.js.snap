// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render nav 1`] = `
<div
  className="nav"
>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/add"
  >
    Add Course
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/addSchool"
  >
    Add School
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/preview"
  >
    Preview
  </NavLink>
</div>
`;
